app/graphql/graphql_router.py

```diff
@@ -64,4 +67,4 @@ class Mutation:
     )
 
 
-schema = strawberry.federation.Schema(Query, mutation=Mutation, types=[User], enable_federation_2=True)
+schema = strawberry.federation.Schema(Query, mutation=Mutation, types=[User, SyndicatedArticle], enable_federation_2=True)
```

Comment:
```suggestion
types = [User, SyndicatedArticle]

schema = strawberry.federation.Schema(Query, mutation=Mutation, types=types, enable_federation_2=True)
```
Let's keep lines at 120 characters or less.